src/lib/components/footer.svelte

```diff
@@ -17,4 +17,8 @@
       >
     </li>
   </ul>
+  <br />
```

Comment:
Best practice is to use css to style the components and not `<br>`.

Reply:
Okay, I'll remember this from next time.